dune-project

```diff
@@ -1,41 +1,26 @@
-(lang dune 2.0)
-
-(using menhir 2.0)
-
+(lang dune 3.17)
 (name html_of_wiki)
 
+(generate_opam_files true)
 (implicit_transitive_deps false)
 
 (license "LGPL-2.1 with OCaml linking exception")
-
 (authors "The ocsigen team <dev@ocsigen.org>")
-
-(maintainers
- "The ocsigen team <dev@ocsigen.org>"
- "Leo Valais <leo.valais97@gmail.com>")
-
-(source
- (github ocsigen/html_of_wiki))
-
-(bug_reports "https://github.com/ocsigen/html_of_wiki/issues")
-
-(homepage "https://github.com/ocsigen/html_of_wiki")
-
+(maintainers "The ocsigen team <dev@ocsigen.org>" "Leo Valais <leo.valais97@gmail.com>")
+(source (github ocsigen/html_of_wiki))
 (documentation "https://ocsigen.org/html_of_wiki/2.0/manual/intro")
 
-(generate_opam_files true)
-
 (package
  (name html_of_wiki)
  (synopsis "A wikicreole to HTML compiler")
- (description
-  "a static website generator for software projects, using wikicreole syntax.")
+ (description "a static website generator for software projects, using wikicreole syntax.")
  (depends
+  (ocaml (< 5.0))
```

Comment:
This seem to be the only significant change in this large diff. Why is it not compatible with ocaml 5 ? It would be more useful to fix the build. The rest of changes don't seem related to fixing the build.  I don't understand the motivation behind all this.